web/modules/custom/ai_screening_project_track/src/Helper/ProjectTrackTypeHelper.php

```diff
@@ -171,6 +173,36 @@ public function validateConfiguration(array $configuration): void {
     }
   }
 
+  /**
+   * Determine the max possible values that a project track can achieve.
+   *
+   * @throws \Drupal\ai_screening_project_track\Exception\InvalidValueException
+   */
+  public function getProjectTrackTypeMaxPossible(Term $projectTrackType): array {
+    $currentMax = [];
+    $webforms = $projectTrackType->get('field_webform')->referencedEntities();
+    // Loop over each tool in the project track.
+    foreach ($webforms as $webform) {
+      $elements = $webform->getElementsDecodedAndFlattened();
+      // Look at each field in the webform.
+      foreach ($elements as $element) {
+        // Currently we only look at weighted radios.
+        if ('ai_screening_weighted_radios' === $element['#type'] && array_key_exists('#options', $element)) {
+          // Get all options for the field.
+          foreach ($element['#options'] as $optionValues => $option) {
+            $values = FormHelper::getIntegers($optionValues);
+            // Get possible max for each element.
+            foreach ($values as $key => $value) {
+              $currentMax[$key] = max($currentMax[$key] ?? 0, $value);
```

Comment:
Shouldn't the values be accumulated, i.e. added together, to get the maximum value for each dimension? If I understand the code correctly, you just get the largest value of any option.

If we have two options, e.g.

```
1,2,4: Yes
2,3,1: No
```

, then I expect the maximum values to be `3,5,5` and not `2,3,4`. I may be missing something …

Reply:
Not quite we don't sum each options value but rather the highest valued option of each question so:

```
**Question1**
1,2,4: Yes
2,3,1: No

**Question2**
4,5,6: Full
3,4,7: Half 
8,4,1: Quarter
4,7,3: Double
```

We should expect ```10,10,11```